### PR TITLE
Updated Docusaurus configs to publish docs website

### PR DESCRIPTION
**Pull request by abhipardhu**
-- Updated Docusaurus configs with website url, sample users etc.
-- Updated docs to render images correctly.
-- Updated Readmes.

Site is at: https://wepay.github.io/waltz/docs/introduction.html

### CODE REVIEW

File: website/siteConfig.js [hunk 1]
```diff
@@ -11,20 +11,20 @@
 // List of projects/orgs using your project for the users page.
 const users = [
   {
-    caption: 'User1',
+    caption: 'WePay',
     // You will need to prepend the image path with your baseUrl
     // if it is not '/', like: '/test-site/img/image.jpg'.
-    image: '/img/undraw_open_source.svg',
-    infoLink: 'https://www.facebook.com',
```
Comment (abhipardhu) [inline website/siteConfig.js]: Didn't link to any pages. Just updated the example here.

File: website/siteConfig.js [hunk 2]
```diff
@@ -11,20 +11,20 @@
...
     pinned: true,
   },
 ];
 
 const siteConfig = {
   title: 'Waltz', // Title for your website.
   tagline: 'An open source quorum-based distributed write-ahead log',
-  url: 'https://your-docusaurus-test-site.com', // Your website URL
-  baseUrl: '/', // Base URL for your project */
+  url: 'https:wepay.github.io', // Your website URL
```
Comment (criccomini) [inline website/siteConfig.js]: Should this have `://` ?

File: website/README.md [hunk 3]
```diff
@@ -1,10 +1,25 @@
...
+For steps 1 and 2, see details [here](https://docusaurus.io/docs/en/installation).
+
+## Testing your docs updates on localhost:
+1. Start the webserver by doing `yarn start`, it will redirect to a published docs page on your default browser.
+
+## Publishing to public docs site:
+1. Make sure that your local master has the intended docs changes to be published.
+2. Run the following script from inside /website directory,
+
+    GIT_USER=<GIT_USER> \ # Preferably, the publisher's git username. It should have push access to the waltz repository.
```
Comment (criccomini) [inline website/README.md]: Nit: spacing here is a bit off

File: docs/application-programming-model.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Read/Write operations are done over the network. An operation may fail at any po
 
 To address the above concerns, We designed Waltz based on a stream oriented communication instead of a RPC based communication.
 
-![RPC communication design](/img/docs/rpc-comm.png)
+![RPC communication design](/waltz/img/docs/rpc-comm.png)
```
Comment (criccomini) [inline docs/application-programming-model.md]: Is this change going to work locally, or will images now not be visible locally?

Comment (abhipardhu) [reply docs/application-programming-model.md]: Do you mean, from the website published to localhost?

File: website/siteConfig.js (hunk 2 above)
Comment (abhipardhu) [reply website/siteConfig.js]: It seems like it is optional.
The succeeding example didn't have an ending `/`, so didn't put it here.

File: docs/application-programming-model.md (hunk 4 above)
Comment (abhipardhu) [reply docs/application-programming-model.md]: Ah! I see what you meant.

Comment (abhipardhu) [reply docs/application-programming-model.md]: These pages https://github.com/wepay/waltz/tree/master/docs ?

(Looks like it is not working on current master, as Biplap pointed out. Let me update the directory structure.)

File: website/siteConfig.js (hunk 2 above)
Comment (abhipardhu) [reply website/siteConfig.js]: Oops.. shoot! I got it! Will update.

**Pull request merged**